tests/integration/test_storage_mysql/test.py

```diff
@@ -114,6 +115,19 @@ def test_table_function(started_cluster):
     conn.close()
 
 
+def test_enum_type(started_cluster):
+    table_name = 'test_enum_type'
+    conn = get_mysql_conn()
+    create_mysql_table(conn, table_name)
+    node1.query('''
+CREATE TABLE {}(id UInt32, name String, age UInt32, money UInt32, source Enum8('IP' = 1, 'URL' = 2)) ENGINE = MySQL('mysql1:3306', 'clickhouse', '{}', 'root', 'clickhouse', 1);
+'''.format(table_name, table_name))
+    node1.query("INSERT INTO {}(id, name, age, money) VALUES (1,'name',0, 0)".format(table_name))
```

Comment:
Is it possible to INSERT `source` (enum column)?

Reply:
let me have a go